### ISSUE TITLE
Critical Worker Timeout in Logs

### ISSUE THREAD

**Issue by conzty01** [conzty01/RA_Scheduler#103]
**Describe the bug**
The following messages were logged in PRODUCTION:
```
 Mar 22 15:31:02Z  [2021-03-22 15:31:02.235] INFO in scheduler4_1: Starting Scheduling Process
 Mar 22 15:31:02Z  [2021-03-22 15:31:02.378] INFO in scheduler4_1: Could Not Generate Schedule
 Mar 22 15:31:02Z  [2021-03-22 15:31:02.384] INFO in schedule: DECREASE LDAT: 4
 Mar 22 15:31:02Z  [2021-03-22 15:31:02.391] INFO in scheduler4_1: Starting Scheduling Process
 Mar 22 15:31:03Z  [2021-03-22 15:31:03.276] INFO in scheduler4_1: Could Not Generate Schedule
 Mar 22 15:31:03Z  [2021-03-22 15:31:03.276] INFO in schedule: DECREASE LDAT: 3
 Mar 22 15:31:03Z  [2021-03-22 15:31:03.279] INFO in scheduler4_1: Starting Scheduling Process
 Mar 22 15:31:32Z  at=error code=H12 desc="Request timeout" method=POST path="//schedule/api/runScheduler" host=ra-scheduler.herokuapp.com request_id=f35cbf1d-f19e-4d43-86b1-adb2bb6e09f6 fwd="x.x.x.x" dyno=web.1 connect=1ms service=30008ms status=503 bytes=0 protocol=https
 Mar 22 15:31:32Z  [2021-03-22 15:31:32 +0000] [4] [CRITICAL] WORKER TIMEOUT (pid:10)
 Mar 22 15:31:32Z  [2021-03-22 15:31:32 +0000] [14] [INFO] Booting worker with pid: 14
 Mar 22 15:31:35Z  /app/.heroku/python/lib/python3.6/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
 Mar 22 15:31:35Z  """)
 Mar 22 15:31:36Z  /app/.heroku/python/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:794: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future. Set it to True or False to suppress this warning.
 Mar 22 15:31:36Z  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 Mar 22 15:38:21Z  at=info method=GET path="/schedule/editSched" host=ra-scheduler.herokuapp.com request_id=6b6ce7f9-c45d-4f13-8254-90d5048f7acd fwd="x.x.x.x" dyno=web.1 connect=1ms service=72ms status=200 bytes=19010 protocol=https
```

**To Reproduce**
Unknown

**Additional context**
The user had just run the scheduler and exported the April 2021 schedule for one hall and switched to do the same for a second hall. However, when they attempted to run the scheduler, it appeared to hang for a minute and upon a refresh of the page, no schedule was generated.

**Comment by conzty01** [comment]
In a quick Google Search, I did find the following GitHub issue for gunicorn:

https://github.com/benoitc/gunicorn/issues/1440

**Comment by conzty01** [comment]
Looks like it could be as simple as a timeout error for Heroku

https://stackoverflow.com/questions/24799347/django-app-in-heroku-getting-worker-timeout-error

**Comment by conzty01** [comment]
This issue should be addressed with PR #106 which moves the scheduling process away from the Web Server and into its own process.
